site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Kubernetes: there is an app for that"
+image: /img/posts/2020/11/19/survey-results-banner.png
+excerpt: Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems
+author_name: Milan Klanjsek
+author_avatar: /img/contributors/milan-klanjsek.jpg
+categories: ['kubernetes']
+tags: ['octant', 'release']
+---
+
+<h1>Kubernetes: there is an app for that</h1>
+
+Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems. The command-line version will still be available for the next couple of releases to allow a smooth transition for everyone,after which we will be 100% focused on the desktop version.
```

Comment:
Comma: `ours, and`

Break into new sentence: `everyone.` Then something to the effect of "Eventually we will be 100% focused on the desktop version".